torch/csrc/jit/script/init.cpp

```diff
@@ -178,11 +178,49 @@ static Self moduleSelf(
   };
 }
 
-static void setInputTensorTypes(Graph& g, const Stack& stack) {
-  AT_ASSERT(stack.size() == g.inputs().size());
-  for (size_t i = 0; i < stack.size(); ++i) {
-    g.inputs().at(i)->setType(
-        DimensionedTensorType::create(stack.at(i).toTensor()));
+static TypePtr getTensorType(
+    const at::Tensor& t,
+    bool isDimensionedTensor) {
```

Comment:
I find it's a bit hard to understand the meaning of `isDimensionedTensor`'s meaning. Can we get a better name for it? And add some comments why we need it.

Reply:
Thanks, changed to `type_kind` instead, marking what is the desired kind of tensor type.